app/views/system_admin/users/index.html.erb

```diff
@@ -26,7 +26,7 @@
                            html: { data: { module: "app-users-autocomplete" } } do |f| %>
       <%= f.govuk_text_field :search,
                              label: { text: "Search for a user", size: "s" },
-                             hint: { text: "Search using the person’s name, email, accredited provider or lead school", size: "s" },
+                             hint: { text: "Search using the person’s name, email, accredited provider or school", size: "s" },
```

Comment:
```suggestion
                             hint: { text: "Search using the person’s name, email, accredited provider or lead partner", size: "s" },
```

It will be lead partner as that is the item associated with the user